### DISCUSSION TITLE
Merge two accounts

### DISCUSSION THREAD

**Discussion by srvkmr130** [community/community#28318] [Other Feature Feedback, Questions, & Ideas]
I have my personal account and another being official account(created with office email -id),
I have two questions mostly -

1. What happens to my official github account created from office email-Id if my office email gets deleted .
2. Is there any way to merge both the accounts (say merge all my contributions , commits etc to my personal account - sounds silly but yeah just wanted to confirm once).

**Comment by airtower-luna** [comment]
> What happens to my official github account created from office email-Id if my office email gets deleted.

If you don't have 2FA set up you probably won't be able to log in any more, and even with 2FA you might run into other trouble if you can't receive relevant mails.

> Is there any way to merge both the accounts (say merge all my contributions , commits etc to my personal account - sounds silly but yeah just wanted to confirm once).

If it's just about contributions: Add the office email address to your private account, and contributions will be assigned to it. Naturally that'd require removing it form the office account first.

For repositories you can [transfer them](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository).
